mne/viz/utils.py

```diff
@@ -1028,8 +1028,6 @@ def _plot_raw_onkey(event, params):
             _setup_annotation_fig(params)
         else:
             params['fig_annotation'].canvas.close_event()
-    elif event.key == 'b':
-        _setup_butterfly(params)
```

Comment:
So butterfly no longer works for Epochs / ICA, right? I guess we can live with it temporarily since those would be the next PRs

Reply:
it never worked for ICA (`_setup_butterfly` is a no-op if it's an ICA object), and Epochs has its own keypress function.  So this isn't even a temporary loss of function

Reply:
(I did this to allow getting rid of `_setup_butterfly_params` entirely, which is otherwise no longer needed)